### PR TITLE
Add objects to represent Books of the Bible

### LINKED ISSUE #15: Create a predefined object for every book of the Bible

### PR DESCRIPTION
**Pull request by nathansit**
Closes #15

### CODE REVIEW

**Comment by nathansit** [comment]
Lint not cooperating
But that will be tomorrow's problem

File: src/objects/booksOfTheBible.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import Book from "../classes/Book";
+
+// thank you Gemini
+// underscore in front of numbers to keep the naming convention consistent
+
+const Genesis = new Book();
```
Comment (gideontong) [inline src/objects/booksOfTheBible.ts]: Please add the lint disabler

File: src/objects/booksOfTheBible.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import Book from "../classes/Book";
+
+// thank you Gemini
```
Comment (gideontong) [inline src/objects/booksOfTheBible.ts]: Please remove this comment

File: src/objects/booksOfTheBible.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+// underscore in front of numbers to keep the naming convention consistent
```
Comment (gideontong) [inline src/objects/booksOfTheBible.ts]: Please change the names to Book_1

**Review by gideontong — CHANGES_REQUESTED** [review]

File: src/objects/booksOfTheBible.ts [hunk 4]
```diff
@@ -0,0 +1,83 @@
+/* eslint @typescript-eslint/no-unused-vars: 0 */
```
Comment (gideontong) [inline src/objects/booksOfTheBible.ts]: Please do not use global disables

**Review by gideontong — CHANGES_REQUESTED** [review]
Almost

**Review by gideontong — CHANGES_REQUESTED** [review]
You haven't made the requested changes. Also while you're at it, please rename the file to `data/books.ts`

**Comment by gideontong** [comment]
While you're at it, please also rename the PR title

File: src/objects/booksOfTheBible.ts (hunk 4 above)
Comment (nathansit) [reply src/objects/booksOfTheBible.ts]: I don't understand. I have dozens of unused variables atm. Would you rather have me use block comments to disable them, or modify the configuration file?

```
/* eslint-disable */

alert('foo');

/* eslint-enable */
```

**Comment by gideontong** [comment]
You didn’t complete any of the requested changes. Going to close this PR tomorrow unless you update it as it’s now empty diff

File: src/objects/booksOfTheBible.ts (hunk 4 above)
Comment (gideontong) [reply src/objects/booksOfTheBible.ts]: Individual disable so that you don’t miss one later

**Comment by gideontong** [comment]
Ok, just one more thing left to do

**Comment by gideontong** [comment]
It's now an empty diff

**Review by gideontong — CHANGES_REQUESTED** [review]
It's now an empty diff

**Comment by gideontong** [comment]
Please change the PR title to the issue title if you can, not a requirement

**Comment by nathansit** [comment]
Bruh I did it again huhEnviado desde mi iPhoneEl jun 27, 2024, a la(s) 01:20, Gideon Tong ***@***.***> escribió:﻿
@gideontong requested changes on this pull request.

It's now an empty diff

**Review by gideontong — APPROVED** [review]

**Pull request merged**